standards/JWT-SVID.md

```diff
@@ -0,0 +1,112 @@
+# The JWT SPIFFE Verifiable Identity Document
+
+## Status of this Memo
+This document specifies an experimental identity document standard for the internet community, and requests discussion and suggestions for improvements. It is a work in progress. Distribution of this document is unlimited.
+
+## Table of Contents
+
+1\. [Introduction](#1-introduction)  
+2\. [JOSE HEader](#2-jose-header)  
+2.1. [Algorithm](#21-algorithm)  
+2.2. [Key ID](#22-key-id)  
+2.3. [Type](#23-type)  
+3\. [JWT Claims](#3-jwt-claims)  
+3.1. [Subject](#31-subject)  
+3.2. [Audience](#32-audience)  
+3.3. [Expiration Time](#33-expiration-time)  
+4\. [Token Signing and Validation](#4-token-signing-and-validation)  
+5\. [Token Transmission](#5-token-transmission)  
+5.1. [Serialization](#51-serialization)  
+5.2. [HTTP](#52-http)  
+5.3. [gRPC](#53-grpc)  
+6\. [Security Considerations](#6-security-considerations)  
+6.1. [Replay Protection](#61-replay-protection)  
+6.2. [Audience](#62-audience)  
+6.3. [Transport Security](#63-transport-security)  
+Appendix A. [Validation Reference](#appendix-a-validation-reference)  
+
+## 1. Introduction
+JWT-SVID is the first token-based SVID in the SPIFFE specification set. Aimed at providing immediate value in solving difficulties associated with asserting identity across Layer 7 boundaries, compatibility with existing applications and libraries is a core requirement.
+
+JWT-SVIDs are standard JWT tokens with a handful of restrictions applied. JWTs have historically proven difficult to get right, gaining a reputation in the security community as a technology which is likely to introduce vulnerabilities in its deployments and implementations. JWT-SVID takes steps to mitigate these problems as much as possible without breaking compatibility with existing applications and libraries.
```

Comment:
Ported comment by @gcallaghan from draft spec:
```This may be slightly contentious. I would argue that the JOSE spec as written has been difficult to implement securely due to some ambiguity and the "crypto agility" as mentioned below. JWT's themselves are just a signed bundle of metadata with some reserved claims.```

Reply:
Good point, I will update the text to clarify this. Thanks for the feedback